Gemfile

```diff
@@ -111,6 +111,7 @@ group :development, :test do
   gem 'nokogiri'
   gem 'overcommit'
   gem 'pry-byebug'
+  gem 'rexml', '3.2.4'
```

Comment:
Is this still needed? I think a fix has been implemented around this now.

Reply:
Once it's merged into master won't it update anyway?